integrations/standalone/src/index.tsx

```diff
@@ -34,14 +34,15 @@ export async function start(): Promise<void> {
 
   const initialize = async (connection: Connection) => {
     const client = await ClientJsonRpc.startClient(connection);
+    const initializeResult = await client.initialize({ app, pmv, file });
```

Comment:
Not sure if we should wait here...
Maybe it is better to not await and add the promise into the editor and in it use the hook "use" to get the await status. with default to false